README.md

```diff
@@ -44,13 +44,13 @@ confluent-kafka-dotnet is distributed via NuGet. We provide five packages:
 To install Confluent.Kafka from within Visual Studio, search for Confluent.Kafka in the NuGet Package Manager UI, or run the following command in the Package Manager Console:
 
 ```
-Install-Package Confluent.Kafka -Version 1.6.0-RC3
+Install-Package Confluent.Kafka -Version 1.6.1-RC2
```

Comment:
I've always wanted to know... Where do these live; is there a private feed, i.e., `1.6.0-rc3` is not on nuget ?

Reply:
sometimes I put them on nuget to facilitate some testing (then hide them once the full release is out).

but there is also a feed on appveyor as documented here:
https://github.com/confluentinc/confluent-kafka-dotnet#branch-builds